subsys/net/l2/openthread/openthread.c

```diff
@@ -441,5 +441,29 @@ static enum net_l2_flags openthread_flags(struct net_if *iface)
 	return NET_L2_MULTICAST;
 }
 
+struct otInstance *openthread_get_instance(void)
```

Comment:
Can there be more than one otInstance in the system?
If yes, how we could select some other instance?
BTW, should we name this `openthread_get_default_instance()` to make it obvious what instance is being selected.

Reply:
That's a good point. Changed to `openthread_get_default_instance`